addons/mail/static/src/components/composer/composer.xml

```diff
@@ -83,6 +83,7 @@
                         t-on-o-composer-suggestion-clicked="_onComposerSuggestionClicked"
                         t-on-o-composer-text-input-send-shortcut="_onComposerTextInputSendShortcut"
                         t-on-paste="_onPasteTextInput"
+                        t-key="composer.localId"
```

Comment:
Maybe this should be on `<Composer/>` itself it that also works? Actually I'm really wondering if we shouldn't do this fix for all of our components that are related to models.

Reply:
on the main div of composer.xml on or each <Composer> node ?

Reply:
That's a good question actually. I was thinking on each node, to ensure we actually get a new component when we change record. But maybe just on the main div would make less noise everywhere. I think the end result will be the same for focus. But it might be different for other cases (e.g. variables we store on the component itself, and especially when we do processing in `_update` with check depending on previous/new record being displayed, those could be removed if we made that change I think).

Reply:
Considering this is a fix in stable, let's just keep it here on textinput where we have an issue. We can try more drastic changes in master. (I think it would be beneficial to investigate this asap)

Reply:
I haven't caught the whole discussion, but I assume something was off with reusing `Composer` component on change of thread cache of thread view, so this enforces change of component.

Ok for this hack. In theory this issue should not exist (stateless components), but whatever.